ui/app/components/pending-tx.js

```diff
@@ -261,6 +264,15 @@ PendingTx.prototype.render = function () {
           }, 'Transaction Error. Exception thrown in contract code.')
         : null,
 
+        !isValidAddress ?
+          h('.error', {
+            style: {
+              marginLeft: 50,
+              fontSize: '0.9em',
+            },
+          }, 'Recipient address is invalid sending this transaction will result in a loss of ETH.')
```

Comment:
> Sending this transaction will result in a loss of ETH.

do we show a warning or do we disallow approval?
if we dont allow this, we dont need to explain the consequences.
> Recipient address is invalid.
should suffice